### DISCUSSION TITLE
Changing keybindings-scrolling using arrow keys

### DISCUSSION THREAD

**Discussion by sstac1** [xournalpp/xournalpp#5731] [Q&A] [answered]
Hello, I'd like to change some keybindings. To be more precise, I'd like to scroll my documents using just up or down arrow instead of ctrl+up or down arrow. I have read it's possible to customize shortcuts using plugins but I haven't found an answer if it's possible to change the pre-defined ones and if it is possible, how to change this one exactly? Thanks in advance!

**Comment by rolandlo** [comment, marked as answer]
I think it's not possible to override up or down arrows.
